dashboard/src/t5gweb/libtelco5g.py

```diff
@@ -191,6 +191,29 @@ def get_case_number(link, pfilter='cases'):
                 return parsed_url.path.split('/')[3]
     return ''
 
+def add_watcher_to_case(case, username, token):
+
+    # Intenal Apps
+    API_URL = 'https://api.enterprise.redhat.com'
```

Comment:
one suggestion: we are at a point where we are making a few different calls to hydra. The FQDN you use here is different than e.g. in cache, but it all to various `hydra` endpoints. I wonder:
1/  if we can move the API FQDN to a configuration variable and use the same one for consistency?
2/ should we make a little helper function for the get token/make headers/requests.get dance?

Reply:
@dcritch 
1/ definetly, I see many "url"s in the cache.py repeating the same. Sounds good moving API FQDN to a conf var. I'll work on that. Note, that there's internal and customer FQDN environments (see hydra_api_curl_examples_case_management), could be useful at some point. 
2/ it would help to simplify  

Also, I'll add a boolean flag in the team variable call "notifieduser" to enable/disable automatic watching,  "jira-user" is the same than "ssoUsername" which helps.

Reply:
new changes:

1.  "t5gweb/libtelco5g.py". 

- Added add_watcher_to_case function updates to add a watcher to a support case when a case is assigned.   
- Added a new field "notifieduser" on the "team" configuration variable to let users being added as a watcher or no.

2.  "t5gweb/cache.py". 
- Replaced all RH Portal API URLs with the new configuration variable 'cfg['redhat_api']'.
- Case details. Added notified_users to case_details so this info is not in the regular cases. 

3.  "t5gweb/utils.py". Added new variable "cfg['redhat_api']" from the os.environment. Also, changes on the config map.

4.  "bin/watcher_case.py". Changed env var from API_OFFLINE_TOKEN to REDHAT_API_TOKEN and fix a issue when getting a sso_usernames from case data. 

5. "build/dashboard.sh". Just a typo fixed.

For the helper we mentioned I'll create an new issue to avoid having a lot of changes here.